VideoWeb/VideoWeb/ClientApp/src/app/waiting-space/private-consultation-room-controls/private-consultation-room-controls.component.scss

```diff
@@ -138,7 +138,7 @@
 .dropdown {
   position: relative;
   display: inline-block;
-  @media only screen and (min-width: $breakpoint-large) {
+  @include responds-to(large) {
```

Comment:
Changes in this file are an example of how this mixin will work. I left other files as is.

Reply:
looks nice, I'm happy if you can keep doing this